addons/binding/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/discovery/SensorDiscoveryService.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.jeelink.discovery;
```

Comment:
You might want to move that to an internal package, e.g. `org.openhab.binding.jeelink.internal.discovery`.